src/Regis/Application/Model/Git/Diff/File.php

```diff
@@ -66,4 +67,23 @@ public function getChanges(): array
     {
         return $this->changes;
     }
+
+    public function findPositionForLine(int $line): int
+    {
+        $changes = $this->getChanges();
+
+        /** @var Change $change */
+        foreach ($changes as $change) {
+            $rangeStart = $change->getRangeNewStart() - 1;
+
+            /** @var Line $diffLine */
+            foreach ($change->getAddedLines() as $diffLine) {
+                if ($rangeStart + $diffLine->getPosition() === $line) {
+                    return $diffLine->getPosition();
+                }
+            }
+        }
+
```

Comment:
Avoid using static access to class '\Regis\Application\Inspection\Exception\LineNotInDiff' in method 'findPositionForLine'.